tests/components/xy-plot-tests.js

```diff
@@ -148,3 +150,18 @@ test('Render a bar chart with some nonAnimatedProps', t => {
 
   t.end();
 });
+
+test('testing flexible charts', t => {
```

Comment:
yeah they're not great though bc the flexible dimension(s), at test time, is going to be 0px. it's animated.